runtime/builder.go

```diff
@@ -20,8 +20,8 @@ type AppBuilder struct {
 
 // DefaultGenesis returns a default genesis from the registered
 // AppModuleBasic's.
-func (a *AppBuilder) DefaultGenesis() map[string]json.RawMessage {
-	return a.app.basicManager.DefaultGenesis(a.app.cdc)
+func (a *AppBuilder) DefaultGenesis() map[string]proto.Message {
```

Comment:
Will we break the AppModule interface again once we remove gogoproto?

How hard would it be to use the `api/` genesis state today?

Reply:
pulsar is not production ready from my understanding. We shouldn't force people to use it yet. the interface break is fairly small in the future, if we document then people will be fine with it